includes/wizards/class-wizard.php

```diff
@@ -247,4 +262,15 @@ public function get_style_dependencies( $dependencies = [] ) {
 	 * @return string The wizard name.
 	 */
 	abstract public function get_name();
+
+	/**
+	 * Load wizard sections.
+	 * 
+	 * @param Section[] $sections Array of Section objects.
+	 */
+	public function load_wizard_sections( $sections ) {
+		foreach ( $sections as $section_slug => $section_class ) {
+			$this->sections[ $section_slug ] = new $section_class();
```

Comment:
Non-blocking question: Is it worth doing any validation here before instantiating?

Reply:
IMO, errors are important to drive engineering. I have made it more presentable.
![Screenshot 2024-06-19 at 23 07 17](https://github.com/Automattic/newspack-plugin/assets/3676975/14862127-ff20-4ea1-91ca-a5be1e88c6fc)